convert-hf-to-gguf.py

```diff
@@ -469,6 +469,9 @@ def get_vocab_base_pre(self, tokenizer) -> str:
         if chkhsh == "27949a2493fc4a9f53f5b9b029c82689cfbe5d3a1929bb25e043089e28466de6":
             # ref: https://huggingface.co/jinaai/jina-embeddings-v2-base-de
             res = "jina-v2-de"
+        if chkhsh == "fcace8b9cac38ce847670c970cd5892031a753a1ef381abd1d9af00f713da085":
+            # ref: https://huggingface.co/microsoft/phi-2
+            res = "phi-2"
```

Comment:
This new pre-tokenizer has to be handled in `llama.cpp`:

https://github.com/ggerganov/llama.cpp/blob/e18bc6aaf3b547890609ed254ee5248e720e5840/llama.cpp#L4414-L4475

Reply:
Thats not necessary. I solved this already in #7219.